translate/storage/mozilla_lang.py

```diff
@@ -116,6 +130,13 @@ def parse(self, lines):
     def serialize(self, out):
         if self.is_active or self.mark_active:
             out.write(b"## active ##\n")
+        for header in self._headers:
```

Comment:
I guess the distinction is headerline (a single line in a header) vs header (a group of items making up a header) vs headers (a number of header blocks).  Tbh I'm not that fussed